web/vite.config.js

```diff
@@ -12,6 +12,9 @@ const upstream = {
 };
 
 export default defineConfig({
+  build: {
+    chunkSizeWarningLimit: 2_000_000,
```

Comment:
2mb would be a really big chunk. It'd probably be nicer if we can figure out how to reduce chunk size instead of silencing this warning

Reply:
That would be great, I think the problem comes from `Photosphere` and `Map`, without these modules the asset-viewer chunk size goes from `1,581.17 kB` to `178.95 kB` (uncompressed)